lib/cli/daemoncommand.cpp

```diff
@@ -367,6 +367,9 @@ static void UmbrellaSignalHandler(int num, siginfo_t *info, void*)
 static void WorkerSignalHandler(int num, siginfo_t *info, void*)
 {
 	switch (num) {
+		case SIGUSR1:
+			// Not ready, yet
```

Comment:
Comment could be better. Something along the lines of "Ignore SIGUSR1 during startup phase. This signal handler will be replaced with the real one in [reference to where it happens]."